pipeline/5/Jenkinsfile-tier-1-master.groovy

```diff
@@ -0,0 +1,98 @@
+/*
+    Pipeline script for executing Tier 1 jobs for RH Ceph 5.0.
+*/
+// Global variables section
+
+def nodeName = "centos-7"
+def cephVersion = "pacific"
+def sharedLib
+def test_results = [:]
+def composeInfo = ""
+def tier1Jobs = ["rhceph-5-tier-1-deploy", "rhceph-5-tier-1-object", "rhceph-5-tier-1-rbd"]
+env.rhcephVersion = "5.0-rhel-8"
```

Comment:
Any particular reason why we are setting this environment variable here?

Reply:
Using withEnv option